devops/jobs/CheckSesLimits.groovy

```diff
@@ -0,0 +1,107 @@
+
+                    wrappers common_wrappers
+                    logRotator common_logrotator
+
+                    wrappers {
+                        credentialsBinding {
+                            file('AWS_CONFIG_FILE','tools-edx-jenkins-aws-credentials')
+                            def variable = "ses-role-${deployment}-arn"
+                            string('ROLE_ARN', variable)
+                        }
+                      }
+
+                    def crit_threshold = extraVars.get('CRIT_THRESHOLD','')
+                    def warn_threshold = extraVars.get('WARN_THRESHOLD','')
+                    def regions = env_config.get('regions')
+
+                    def config_internal_repo = "git@github.com:edx/${deployment}-internal.git"
+                    def config_secure_repo = "git@github.com:edx-ops/${deployment}-secure.git"
+
+                    extraVars['CONFIGURATION_INTERNAL_REPO'] = config_internal_repo
+                    extraVars['CONFIGURATION_SECURE_REPO'] = config_secure_repo
+
+                    if (environment == 'prod'){
+                        extraVars['NOTIFY_ON_FAILURE'] = 'tools-edx-jenkins-alert@edx.opsgenie.net'
+                    }
+                    else{
+                        extraVars['NOTIFY_ON_FAILURE'] = 'devops+non-critical@edx.org'
+                    }
+
+                    properties {
+                        rebuild {
+                            autoRebuild(false)
+                            rebuildDisabled(false)
+                        }
+                    }
+
+                    throttleConcurrentBuilds {
+                        maxPerNode(0)
+                        maxTotal(0)
+                    }
+
+                    triggers {
+                        cron("H/10 * * * *")
+                    }
+
+                    parameters common_parameters(extraVars)
+
+                    multiscm common_multiscm(extraVars)
+
+                    environmentVariables {
+                        env('ENVIRONMENT', environment)
+                        env('DEPLOYMENT', deployment)
+                        env('CRIT_THRESHOLD', crit_threshold)
+                        env('WARN_THRESHOLD', warn_threshold)
+                        env('REGIONS', regions)
+                    }
+
+                    steps {
+                        virtualenv {
+                            pythonName('System-CPython-3.5')
+                            nature("shell")
+                            systemSitePackages(false)
+
+                            command(
+                                dslFactory.readFileFromWorkspace("devops/resources/check-ses-limits.sh")
+                            )
+
+                        }
+
+                    }
+
+                    publishers {
+                        mailer(extraVars.get('NOTIFY_ON_FAILURE','devops+non-critical@edx.org'), false, false)
```

Comment:
You have a conditional above that alwas sets this key in the extraVars dict, so I don't think you need a default here.  You should just assert that the NOTIFY_ON_FAILURE key is in the dict.